src/frontend/web_application/webpack/webpack.config.browser.js

```diff
@@ -10,6 +10,7 @@ const isProd = process.env.NODE_ENV === 'production';
 
 let config = Object.assign(baseConfig.getBase('browser'), {
   entry: [
+    'babel-polyfill',
```

Comment:
yes. 
I know, easy to add & hard to remove. 
Right now, I prefer spend time on features than fixing every browser (that will be done while beta-phase).